package.json

```diff
@@ -9,7 +9,7 @@
     "dist"
   ],
   "scripts": {
-    "test": "ts-node --esm src/tests/test.ts",
```

Comment:
For some reason the `--esm` flag fails with Node v20; however without the flag this works fine on all Node versions in the matrix.